src/swap/defi/thorchain.ts

```diff
@@ -213,11 +214,14 @@ export function makeThorchainPlugin(
 ): EdgeSwapPlugin {
   const { io, log } = opts
   const { fetch } = io
-  const { thorname, affiliateFeeBasis } = asInitOptions(opts.initOptions)
+  const { thorname, affiliateFeeBasis, ninerealmsClientId } = asInitOptions(
+    opts.initOptions
+  )
   const affiliateFee = div(affiliateFeeBasis, '10000', DIVIDE_PRECISION)
 
   const headers = {
-    'Content-Type': 'application/json'
+    'Content-Type': 'application/json',
+    'x-client-id': ninerealmsClientId
```

Comment:
These headers are being sent to both thorswap and midgard servers. Do they both care about `ninerealmsClientId`?

Reply:
They don't but thorswap just ignores it.